geiger/src/graph/Graph.js

```diff
@@ -129,6 +144,10 @@ export default class Graph extends Events {
     this._shouldRedraw();
   }
 
+  /**
+   * Remove an edge to the graph by its id.
+   * @param {!string} edgeId
```

Comment:
Let's define a data type for these UUIDs. I am using `GraphNodeId` in my JS code which extends `string`. I suggest you to do the same.

Reply:
Hmm. This is an edge not a node. I'll have to think about it a bit.